### DISCUSSION TITLE
No On/Off state change when I Start the AppleTV with the original remote Control

### DISCUSSION THREAD

**Discussion by DickeBerta** [maxileith/homebridge-appletv-enhanced#139] [Q&A]
Hi,
I tried to use the ON/Off state from the appleTV to start an automation. My wife always take the original apple tv remote control and when she did that, the on/off state will not change in the Home App. This state only change, when I do that in the Home App. 

Can anybody help me? 

Thanks

**Comment by maxileith** [comment]
Hi,

are you using an AirPlay device as your default speaker, e.g. a HomePod?

**Reply by DickeBerta** [reply]
Hi,
no I'm using my TV as default speaker.

**Reply by maxileith** [reply]
In that case, please open a bug report.
